dockstore-common/src/main/scala/io/dockstore/common/Bridge.scala

```diff
@@ -189,15 +192,22 @@ class Bridge(basePath : String) {
   @throws(classOf[WdlParser.SyntaxError])
   def getCallsToDockerMap(file: JFile): util.LinkedHashMap[String, String] = {
     val lines = scala.io.Source.fromFile(file).mkString
-    val ns = WdlNamespaceWithWorkflow.load(lines, Seq(resolveHttpAndSecondaryFiles _)).get
-    val tasks = new util.LinkedHashMap[String, String]()
-
-    val allTasks = findCallToTasks(ns)
-    allTasks.foreach(taskTuple => {
-      val dockerAttributes = taskTuple._2.runtimeAttributes.attrs.get("docker")
-      tasks.put("dockstore_" + taskTuple._1.unqualifiedName, if (dockerAttributes.isDefined) dockerAttributes.get.collectAsSeq(passthrough).map(x => x.toWdlString.replaceAll("\"", "")).mkString("") else null)
-    })
-    tasks
+    try {
+      val ns = WdlNamespaceWithWorkflow.load(lines, Seq(resolveHttpAndSecondaryFiles _)).get
+      val tasks = new util.LinkedHashMap[String, String]()
+
+      val allTasks = findCallToTasks(ns)
+      allTasks.foreach(taskTuple => {
+        val dockerAttributes = taskTuple._2.runtimeAttributes.attrs.get("docker")
+        tasks.put("dockstore_" + taskTuple._1.unqualifiedName, if (dockerAttributes.isDefined) dockerAttributes.get.collectAsSeq(passthrough).map(x => x.toWdlString.replaceAll("\"", "")).mkString("") else null)
+      })
+      tasks
+    } catch {
+      case ex: NoSuchMethodException =>
+        //FIXME: the best we can do is be generous and assume that unknown methods are WDL 1.0 methods until we update
+        // https://github.com/ga4gh/dockstore/issues/2139
+      null
```

Comment:
Return an empty LinkedHashMap instead? It seems like you'll get an NPE later in `mapConverterToToolInfo`? Or does the code handle an NPE somewhere further up the chain and that is OK?